core/platform/models.py

```diff
@@ -359,16 +380,21 @@ class Registry(python_utils.OBJECT):
 
     @classmethod
     def _get(cls):
+        # type: () -> Type[_Gae]
         """Returns the appropriate interface class for platform-specific
         imports.
 
         Returns:
             class. The corresponding platform-specific interface class.
         """
-        return cls._PLATFORM_MAPPING.get(GAE_PLATFORM)
+        klass = cls._PLATFORM_MAPPING.get(GAE_PLATFORM)
+        # Ruling out the possibility of None for mypy type checking.
+        assert klass is not None
+        return klass
 
     @classmethod
     def import_models(cls, model_names):
+        # type: (List[Text]) -> Tuple[ModuleType,...]
```

Comment:
According to the docstring, it should return a list? Also, what's the 2nd param `...`?

Reply:
Tuple[Type, ...] means Tuple of variables of type 'Type'. I think the docstring here is wrong. If we take a look at the code, we see that this calls `Platform.import_models()` and this returns tuple.

Reply:
Done